css/pre_marriage.css

```diff
@@ -1,15 +1,7 @@
-body{
-background-color: #fff; 
-background-image: 
-linear-gradient(90deg, transparent 79px, #abced4 79px, #abced4 81px, transparent 81px),
-linear-gradient(#eee .1em, transparent .1em);
-background-size: 100% 1.2em;
+.beneath-nav {
+	padding-top: 50px;
 }
 
-h1{
-    text-align: center;
-}
-
-h2{
-    text-align: center;
+em{

```

Comment:
"em" is not a good name to use as a style. It's also a unit. Please choose something more descriptive